frontend/src/features/mapSideBar/tabs/propertyAssociations/AssociationContent.tsx

```diff
@@ -0,0 +1,87 @@
+import { ColumnWithProps, renderDate, Table } from 'components/Table';
+import { Api_PropertyAssociation } from 'models/api/Property';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { CellProps } from 'react-table';
+
+interface IPimsInfo {
+  id: string;
+  linkUrl: string;
+  fileIdentifier: string;
+  fileName: string;
+  createdBy: string;
+  createdDate: string;
+  status: string;
+}
+
+export interface IAssociationContentProps {
+  associationName: string;
+  aquisitionFiles?: Api_PropertyAssociation[];
+  linkUrlMask: string;
+}
+
+const AssociationContent: React.FunctionComponent<IAssociationContentProps> = props => {
+  const noDataMessage = `There are no ${props.associationName} files availiable`;
+  if (props.aquisitionFiles === undefined) {
+    return <>{noDataMessage}</>;
```

Comment:
This seems odd, this will prevent the table below from displaying at all, so then why does the table have a noDataMessage as well?

Reply:
This is because if the passed property is undefined it will show the message. And if the property is not undefined but an empty list it will allow the table to handle it, showing the message